webapp/app/forms/fields.py

```diff
@@ -67,6 +67,21 @@ def __call__(self, *args, **kwargs):
         return super().__call__(**kwargs)
 
 
+class SteuerlotseIbanField(SteuerlotseStringField):
+
+    def __call__(self, *args, **kwargs):
+        if 'class' in kwargs:
+            kwargs['class'] = kwargs['class'] + ' iban-input'
+        else:
+            kwargs.setdefault('class', 'iban-input')
```

Comment:
Why `setdefault`? Why not `kwargs['class'] = 'iban-input'`?

Reply:
No particular reason other than I've added the conditional afterwards I've changed it back. This should be exchanged through the potential `_add_classes_to_kwargs `  mentioned in #119 anyway.